### DISCUSSION TITLE
Customizable line spacing for the article list view

### DISCUSSION THREAD

**Discussion by cayubal** [ViennaRSS/vienna-rss#1744] [Ideas]
I'm changing from NetNewsWire to Vienna and so far I'm loving it!

The only thing that's buggin me is the line spacing of the Article List (when "Layout" is set to "Horizontal").
One can change the article list font in the preferences. 
It would be great if one could change the line spacing too!

![vienna_article_list](https://github.com/ViennaRSS/vienna-rss/assets/127452689/74c055df-88c3-4adc-9c7e-b338413540d1)

Bonus points for the possibility to turn off the alternating background ;)

**Comment by Wellenbrett** [comment]
I have been using Vienna for years (actually decades) and appreciate it very much. Compared to Netnewswire, what I appreciate about Vienna is the ability to manually sort my RSS feeds, create subfolders in the feed list, and the German language support. The only thing I like better about Netnewswire is the appearance of the article list and the display of the articles. For comparison, I'm posting screenshots of the Vienna feed as it appears in Vienna and as it appears in Netnewswire. I find the Netnewswire display clearer and the content easier to grasp intuitively. Unfortunately, the article display in Vienna is not theme-capable (as far as I can tell). Netnewswire is open source, so the display could be quickly replicated. But the display of the article list in Vienna cannot be changed. I would very much like to see a standard display in Vienna like in Netnewswire.

Netnewswire:
<img width="1103" height="887" alt="Netnewswire" src="https://github.com/user-attachments/assets/f05d2ecf-922e-4303-91c5-37098d9f1872" />

Vienna:
<img width="1102" height="772" alt="Vienna" src="https://github.com/user-attachments/assets/acfb0651-2594-4275-8f6a-ee499fead6b7" />
